JobConfig/validation/stoppedMuonsSingleStageMT.fcl

```diff
@@ -0,0 +1,4 @@
+#include "JobConfig/validation/stoppedMuonsSingleStage.fcl"
```

Comment:
This is a new file that does not appear to be a part of the big rename.  Is this supposed to be part of it?  I have no objections - just checking.

Reply:
Yes, it is slightly off topic, but likely a useful addition for a future use.